source/reference/command/flushRouterConfig.txt

```diff
@@ -12,13 +12,13 @@ flushRouterConfig
 
 .. dbcommand:: flushRouterConfig
 
-   :dbcommand:`flushRouterConfig` clears the current cluster
-   information cached by a :binary:`~bin.mongos` instance and reloads all
-   :term:`sharded cluster` metadata from the :term:`config database`.
+   :dbcommand:`flushRouterConfig` clears the cached routing table in a
+   :binary:`~bin.mongos` instance. Subsequent :binary:`~bin.mongos`
+   commands update the routing table as routes are needed.
```

Comment:
Subsequent commands against that :binary:`~bin.mongos` will repopulate the routing table cache from the config server as they execute.